src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+function norm(salary) {
+  return +salary.split(',').join('').slice(1);
+}
+
+function sort(l) {
+  const childrenCopy = [...l.children];
+
+  childrenCopy.sort((a, b) => {
+    return norm(b.dataset.salary) - norm(a.dataset.salary);
+  });
+
+  childrenCopy.forEach((emp) => l.append(emp));
+}
+
+function getEmps(l) {
```

Comment:
do not use short forms of words

even in the task description, there is the name of function you have to create - `getEmployees`
generally clear and self-explanatory names are very important